pkg/controller/flow/reconcile.go

```diff
@@ -350,3 +357,41 @@ func (r *reconciler) NewControllerRef(flow *v1alpha1.Flow) *metav1.OwnerReferenc
 	revRef.Controller = &isController
 	return revRef
 }
+
+const (
+	// controllerConfigMapName is the name of the configmap in the eventing
+	// namespace that holds the configuration for this controller.
+	controllerConfigMapName = "flow-controller-config"
+
+	// defaultClusterBusConfigMapKey is the name of the key in this controller's
+	// ConfigMap that contains the name of the default cluster bus for the flow
+	// controller to use.
+	defaultClusterBusConfigMapKey = "default-cluster-bus"
+
+	// fallbackClusterBusName is the name of the cluster bus that will be used
+	// for flows if the controller's configmap does not exist or does not
+	// contain the 'default-cluster-bus' key.
+	fallbackClusterBusName = "stub"
+)
+
+// getDefaultClusterBus returns the value of the 'default-cluster-bus' key in
```

Comment:
This needs to be updated with the new function name

Reply:
whoops!